packages/base/Avatar/src/Avatar/story/ToptalLogo.example.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Avatar, Container } from '@toptal/picasso'
+import { SPACING_4 } from '@toptal/picasso-utils'
+
+const Example = () => (
+  <div>
+    <Container inline>
+      <Avatar
+        size='large'
+        alt='Jacqueline Roque. Pablo Picasso, 1954'
+        name='Jacqueline Roque'
+        src='./jacqueline-with-flowers-1954-square.jpg'
+        emblem
+      />
+    </Container>
+    <Container inline left={SPACING_4}>
+      <Avatar size='medium' name='Jacqueline Roque' emblem />
+    </Container>
+    <Container inline left={SPACING_4}>
+      <Avatar size='small' emblem />
+    </Container>
```

Comment:
```suggestion
    <Container inline left={SPACING_4}>
      <Avatar size='small' emblem />
    </Container>
    <Container inline left={SPACING_4}>
      <Avatar size='xsmall' emblem />
    </Container>
```

Just a "nice to have" suggestion, but I think it would be cool to also demo that on sizes below `small` the logo will not be visible